script/build-authors.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
`/bin/bash` is actually guaranteed to be there per Linux Standard Base.
`/usr/bin/env` is not.

There's various arguments either way for most things, and most scripts engines it can make sense. However, `/bin/bash` and `/bin/sh` are two that have to be around for a basic system to function and can be directly relied upon so they don't make sense to export out to `/usr/bin/env`.

$0.02

Reply:
Hmm. I can look this up later, I suppose. Anyway, thanks for approving the PR